python/ray/train/huggingface/transformers/_transformers_utils.py

```diff
@@ -230,13 +230,21 @@ def on_train_end(self, args, state, control, **kwargs):
 class RayTrainReportCallback(TrainerCallback):
     """A simple callback to report checkpoints and metrics to Ray Tarin.
 
-    This callback overrides the `TrainerCallback.on_save()` method. After
+    This callback is a subclass of `transformers.TrainerCallback
+    <https://huggingface.co/docs/transformers/main/en/main_classes/callback#transformers.TrainerCallback>`_
+    and overrides the `TrainerCallback.on_save()` method. After
     a new checkpoint get saved, it fetches the latest metric dictionary
     from `TrainerState.log_history` and reports it with the latest checkpoint
     to Ray Train.
 
-    If you want more customized reporting logics, please implement your own
-    callbacks following the Transformers integration user guides.
+    Checkpoints will be saved in the following structure:
+
+    checkpoint_00000*/   # Ray Train Checkpoint
+    └─ checkpoint/       # Hugging Face Transformers Checkpoint
```

Comment:
This [doesn't seem to](https://anyscale-ray--40174.com.readthedocs.build/en/40174/train/api/doc/ray.train.lightning.RayTrainReportCallback.html#ray.train.lightning.RayTrainReportCallback) render properly, maybe we need to wrap in a code block?

Reply:
Fixed now!